Documentation/ExtensionArchitecture/FileStructure/ExtLocalconf.rst

```diff
@@ -1,21 +1,24 @@
-.. include:: /Includes.rst.txt
-.. index::
-   File; EXT:{extkey}/ext_localconf.php
-.. _ext-localconf-php:
+..  include:: /Includes.rst.txt
+..  index::
+    File; EXT:{extkey}/ext_localconf.php
+..  _ext-localconf-php:
 
-=========================
-:file:`ext_localconf.php`
-=========================
+===================
+`ext_localconf.php`
+===================
 
-*-- optional*
+..  typo3:file:: ext_localconf.php
+    :scope: extension
+    :regex: /^.*ext\_localconf\.php$/
+    :shortDescription: Should contain additional configuration of $GLOBALS['TYPO3_CONF_VARS'].
 
-:file:`ext_localconf.php` is always included in global scope of the script,
-in the frontend, backend and CLI context.
+    :file:`ext_localconf.php` is always included in global scope of the script,
+    in the frontend, backend and CLI context.
 
-It should contain additional configuration of :php:`$GLOBALS['TYPO3_CONF_VARS']`.
+    It should contain additional configuration of :php:`$GLOBALS['TYPO3_CONF_VARS']`.
 
-This file contains hook definitions and plugin configuration. It must
-not contain a PHP encoding declaration.
+    This file contains hook definitions and plugin configuration. It must
+    not contain a PHP encoding declaration.
```

Comment:
Do you have an idea what is ment here? `header()` is HTML encoding. `BOM` is file encoding. I don't get PHP encoding in context of ext_localconf.php.

Reply:
No idea, I just thought to keep it as it would be out of context to change such information within this PR